test/evp_test.c

```diff
@@ -1989,7 +1989,7 @@ static int kdf_test_init(EVP_TEST *t, const char *name)
     EVP_KDF *kdf;
 
 #ifdef OPENSSL_NO_SCRYPT
-    if (strcmp(name, "scrypt") == 0) {
+    if (strcmp(name, "id-scrypt") == 0) {
```

Comment:
pkey_kdf_test_init also has a check but maybe that one works?

Reply:
Would it be better to standardise on one form or the other? It seems strange that evppkey_kdf.txt calls it "scrypt" but evpkdf.txt calls it "id-scrypt".

Reply:
That would be better. But from my understandings, that could only happen after aliases are supported...